akka-stream/src/main/scala/akka/stream/javadsl/Flow.scala

```diff
@@ -934,6 +934,48 @@ final class Flow[-In, +Out, +Mat](delegate: scaladsl.Flow[In, Out, Mat]) extends
     new Flow(delegate.alsoToMat(that)(combinerToScala(matF)))
 
   /**
+   * Interleave represents deterministic merge of the given [[Source]] with elements of this [[Flow]].
+   * It takes `request` number of elements from this flow to emit downstream, then - same amount for given source,
+   * then repeat process from the beginning.
```

Comment:
Perhaps:

It first emits `request` number of elements from this flow to downstream, then - same amount for `that` source, then repeat process.